swagger.json

```diff
@@ -0,0 +1,421 @@
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/SeaerchType"
+            }
+          },
+          {
+            "name": "source",
+            "in": "query",
+            "description": "Source to search in. { G: Guru Granth Sahib Ji, D: Dasam Granth Sahib, B: Bhai Gurdas Ji Vaaran, A: Amrit Keertan, S: Bhai Gurdas Singh Ji Vaaran N: Bhai Nand Lal Ji Vaaran, R: Rehatnamas & Panthic Sources }",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/Source"
+            }
+          },
+          {
+            "name": "ang",
+            "in": "query",
+            "description": "Which ang to return. Required if searchtype is 5",
+            "required": false,
+            "type": "number"
+          },
+          {
+            "name": "raag",
+            "in": "query",
+            "description": "Which raag to limit search to.",
+            "required": false,
+            "type": "number"
+          },
+          {
+            "name": "writer",
+            "in": "query",
+            "description": "Which writer to limit search to.",
+            "required": false,
+            "type": "number"
+          },
+          {
+            "name": "page",
+            "in": "query",
+            "description": "Pagination param. Which page to return.",
+            "required": false,
+            "type": "number"
+          },
+          {
+            "name": "results",
+            "in": "query",
+            "description": "Pagination param. How many results per page to return.",
+            "required": false,
+            "type": "number"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation"
+          }
+        }
+      }
+    },
+    "/shabads/{shabadID}": {
+      "get": {
+        "summary": "Shabads by ID",
+        "description": "Get an entire Shabad from its ShabadID. More than one shabadID can be specificed with operators '+' or ','",
+        "parameters": [
+          {
+            "name": "shabadID",
+            "in": "path",
+            "description": "Shabad id(s)",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation"
+          },
+          "400": {
+            "description": "Malformed URL"
+          }
+        }
+      }
+    },
+    "/angs/{pageNo}/{sourceID}": {
+      "get": {
+        "summary": "Ang",
+        "description": "Get an entire Ang(s) by number and optional source. More than one Ang can be specificed with operators '+' and '-'",
+        "parameters": [
+          {
+            "name": "pageNo",
+            "in": "path",
+            "description": "Ang number and/or range",
```

Comment:
I don't think this takes both a number and range, so should this be "Ang number or range"?

Reply:
6+12-13 is possible. I'll include some examples.